CMakeLists.txt

```diff
@@ -220,6 +220,8 @@ add_library(libwabt STATIC
   ${AST_PARSER_GEN_C}
   src/type-checker.cc
   src/validator.cc
+  src/stream.cc
+  src/writer.cc
```

Comment:
Oh, just random. I move stuff around when I'm compiling so I get errors earlier. Probably should just alphabetize.